.github/workflows/ci.yml

```diff
@@ -342,13 +355,6 @@ jobs:
         env:
           TAG: ${{ needs.build-edu.outputs.tag }}
           TIMESTAMP: ${{ steps.get_timestamp.outputs.timestamp }}
-      - run: |
-          echo SERVERLESS_SERVICE=edu >> $GITHUB_ENV; 
-          if [[ $GITHUB_REF == $STAGING_BRANCH ]]; then
-            echo SERVERLESS_STAGE=staging >> $GITHUB_ENV;
-          elif [[ $GITHUB_REF == $PRODUCTION_BRANCH ]]; then
-            echo SERVERLESS_STAGE=production >> $GITHUB_ENV;
-          fi
```

Comment:
i see that several of such code blocks have been shifted up. was this necessary for something? 

curious because i notice that you have pushed to `edge` branch several times recently, but our staging lambdas have not been updated in 5 days. wanted to make sure that the serverless deployment pipeline did not break 😂

Reply:
I want to re-use the SERVERLESS_STAGE in DD_ENV.
I don't think this affects the lambda but I will check

Reply:
Made some dummy change to the serverless scripts, it's deployed fine. https://ap-southeast-1.console.aws.amazon.com/lambda/home?region=ap-southeast-1#/functions